app/components/primer/popover_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render Primer::BaseComponent.new(**@kwargs) do %>
+  <%= render message.component do %>
+    <% if heading %>
+      <%= render heading.component do %>
+        <%= heading.content %>
+      <% end %>
+    <% end %>
+    <%= message.content %>
+    <% if button %>
```

Comment:
I find myself not setting this button slot but rather including a `<form>` with a button inside `message`. I may remove this slot.